admin/server/organizations.go

```diff
@@ -1198,6 +1298,264 @@ func (s *Server) ListPublicBillingPlans(ctx context.Context, req *adminv1.ListPu
 	}, nil
 }
 
+func (s *Server) ListOrganizationBillingErrors(ctx context.Context, req *adminv1.ListOrganizationBillingErrorsRequest) (*adminv1.ListOrganizationBillingErrorsResponse, error) {
```

Comment:
Since this file is getting massive, can we break the billing-related RPCs into a separate `admin/server/billing.go` file?